sw/airborne/modules/computer_vision/opticflow_module.c

```diff
@@ -128,7 +128,7 @@ void opticflow_module_run(void)
                            opticflow_result.div_size,
                            opticflow_state.agl);
     //TODO Find an appropiate quality measure for the noise model in the state filter, for now it is tracked_cnt
-    if (quality > 0.8) {
+    if (quality < 0.8) {
```

Comment:
For the future, quality is not the best term here (at least not for how it's used)... it's actually the noise (so lower is better), but for quality you would expect higher to be better...